src/main/java/org/jalasoft/moi/model/Execute.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ *
+ */
+
+package org.jalasoft.moi.model;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class Execute {
+    String command;
+    String salida;
+
+     /**
+     * The contructor recieves the string command
+     */
+    public Execute(String comm){
+        command=comm;
+        salida = null;
```

Comment:
Please the class should be recibe a Language instead of only the command in order to avoid instantiate the execute class in every language.

Reply:
It's not very common initialize variables with `null ` value. If you don't want give it any value just keep your declarations in `String salida;` this initialize your variable with the default value of the String (null).